src/apb/engine.py

```diff
@@ -208,9 +248,25 @@ def cmdrun_init(**kwargs):
     bindable = kwargs['bindable']
     async = kwargs['async']
     params = kwargs['params']
+    skip = kwargs['skip'] or []
 
     description = "This is a sample application generated by apb init"
 
+    # Ensure that command line options are acceptable
+    for skip_option in skip:
```

Comment:
Instead of validating the skip params here, you could use choices in argparse to limit what the user can enter.
https://docs.python.org/3/library/argparse.html#choices